spec/system/admin/products_v3/actions_spec.rb

```diff
@@ -259,20 +259,6 @@ def expect_other_columns_visible
             expect(input_content).to match /COPY OF Apples/
           end
         end
-
-        it "shows error message when cloning invalid record" do
-          # Existing product is invalid:
-          product_a.update_columns(name: nil)
-
-          click_product_clone "Apples"
-
-          expect(page).to have_content "Unable to clone the product"
-
-          within "table.products" do
-            # Products does not include the cloned product.
-            expect(all_input_values).not_to match /COPY OF Apples/
-          end
-        end
```

Comment:
`±` I think we need to keep this. Inevitably we will have invalid products in the system (eg what if validation rules change in the future).

Reply:
I can't think of a way to get this to work .Even if you mock the product to return a `nil` name, the `ProductDuplicator` will set the name to "COPY OF " meaning it will create a valid cloned product. Any ideas ?